### PR TITLE
add pypng from pip

### PR DESCRIPTION
**Pull request by fmessmer**
as available here: https://pypi.org/project/pypng/

### CODE REVIEW

**Comment by ipa-fxm** [comment]
thanks for the review...I updated the PR

package is available for
gentoo: https://packages.gentoo.org/packages/dev-python/pypng

but not for the other variants :point_right: keeping pip

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2516,6 +2516,37 @@ python-pynmea2:
...
+  debian: [python-png]
+  fedora: [python-pypng]
+  gentoo: [dev-python/pypng]
+  ubuntu:
+    artful: [python-png]
+    bionic: [python-png]
+    trusty:
+      pip:
+        packages: [pypng]
+    utopic:
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: We usually don add new keys for EOL distributions. Can you please keep only trusty and xenial here?

For future proofness, we can use the new wildcard syntax to cover upcoming distributions as wel like cosmic:
```
ubuntu:
  '*': [python-png]
  trusty:
    pip:
      packages: [pypng]
  xenial:
    pip:
      packages: [pypng]
```

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2516,6 +2516,37 @@ python-pynmea2:
   ubuntu:
     pip:
       packages: [pynmea2]
+python-pypng:
+  arch:
+    pip:
+      packages: [pypng]
+  debian: [python-png]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: This is only available on buster, we need to provide pip entries for the active distributions that don;t have that package: `jessie` and `stretch`

**Comment by ipa-fxm** [comment]
I hope everything looks good now

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks @ipa-fxm !

**Pull request merged**